src/components/PopularCoursesLayout/PopularCoursesContainer/PopularCard/PopularCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { StyledLinkMore, StyledPopularCardDescription, StyledPopularCardHeader, StyledPopularCardImage, StyledPopularTags, StyledYellowButton } from './styledPopularCard'
+import { StyledFlex } from '../../../../styledConstants'
+import { StyledPopularHorizontalContainer } from '../StyledPopularCourses'
+import { NavLink } from 'react-router-dom'
+import ArrowRight from '../../../../assets/icons/ArrowRight'
+
+interface IPopular {
+  header: string,
+  type: "Design" | "Data Science" | "Programming",
+  countLessons: number,
+  description: string,
+  imageSrc: string,
+  isVertical?: boolean
+}
+const PopularCard = ({header, type, countLessons, description, imageSrc, isVertical}: IPopular) => {
```

Comment:
You shouldn't mix commits together, if you want to fix something in your previous commit, do it in your previous PR.